hogehoge5.rb

```diff
@@ -0,0 +1,25 @@
+
+$id = 0
+
+puts "hello,world"
+
+
+if $id < 0
+  puts "hogehoge"
+end
+
+class Hoge
+  
+  def initialize
+    aaa = 1
+    
+    if aaa < 0
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.<br>Favor modifier if usage when having a single-line body. Another good alternative is the usage of control flow &&/||.